src/test/java/BidTest.java

```diff
@@ -1,2 +1,45 @@
-package PACKAGE_NAME;public class BidTest {
+import models.*;
```

Comment:
its common to mirror the package structure of the src package for tests so create console and models and put respective tests in those packages